src/main/kotlin/com/replaymod/gradle/preprocess/PreprocessTask.kt

```diff
@@ -246,15 +251,35 @@ class CommentPreprocessor(private val vars: Map<String, Int>) {
             n++
             val trimmed = line.trim()
             val mapped = if (trimmed.startsWith(kws.`if`)) {
```

Comment:
Any changes in the `if` block must also be applied to the `ifdef` block (though since https://github.com/ReplayMod/preprocessor/commit/d560a2ec9937da6aa2a4d2a1e9702025107605c8 it's not particularly useful and could probably be removed eventually, but it still exists at the moment).